build failed ltcsuite unknown revision

### Background
build fails when following readme guide to install.
### Your environment

ubuntu 18.04

### Steps to reproduce

go get -d github.com/lightningnetwork/lnd
cd $GOPATH/src/github.com/lightningnetwork/lnd
make && make install

### Expected behaviour

build success

### Actual behaviour

go: github.com/ltcsuite/ltcd@v0.0.0-20180701083835-cdab10132e8c: unknown revision cdab10132e8c
go: error loading module requirements
Makefile:99: recipe for target 'build' failed
make: *** [build] Error 1

I am having the same problem

```

john@T800:/usr/lib$ cd $GOPATH/src/github.com/lightningnetwork/lnd
john@T800:~/gocode/src/github.com/lightningnetwork/lnd$ make
 Building debug lnd and lncli.
GO111MODULE=on go build -v -tags="dev" -o lnd-debug -ldflags "-X github.com/lightningnetwork/lnd/build.Commit=v0.5.1-beta-256-g089d461552ffc05d7b1c22d603af0fa82e788e85" github.com/lightningnetwork/lnd
go: finding github.com/ltcsuite/ltcd v0.0.0-20180701083835-cdab10132e8c
go: github.com/ltcsuite/ltcd@v0.0.0-20180701083835-cdab10132e8c: unknown revision cdab10132e8c
go: error loading module requirements
Makefile:99: recipe for target 'build' failed
make: *** [build] Error 1
john@T800:~/gocode/src/github.com/lightningnetwork/lnd$ GO111MODULE=on go get golang.org/x/sync@42b317875d0fa942474b76e1b46a6060d720ae6e
go: finding github.com/ltcsuite/ltcd v0.0.0-20180701083835-cdab10132e8c
go: github.com/ltcsuite/ltcd@v0.0.0-20180701083835-cdab10132e8c: unknown revision cdab10132e8c
go: error loading module requirements
john@T800:~/gocode/src/github.com/lightningnetwork/lnd$ git version
git version 2.17.1
john@T800:~/gocode/src/github.com/lightningnetwork/lnd$ go version
go version go1.11.4 linux/amd64
```

same here: 
Ubuntu 17.10
go version go1.11.4 linux/amd64
on the latest master code 089d461552ffc05d7b1c22d603af0fa82e788e85
`make && make install
go: finding github.com/ltcsuite/ltcd v0.0.0-20180701083835-cdab10132e8c
go: github.com/ltcsuite/ltcd@v0.0.0-20180701083835-cdab10132e8c: unknown revision cdab10132e8c
go: error loading module requirements
go: finding github.com/ltcsuite/ltcd v0.0.0-20180701083835-cdab10132e8c
go: github.com/ltcsuite/ltcd@v0.0.0-20180701083835-cdab10132e8c: unknown revision cdab10132e8c
go: error loading module requirements
 Building debug lnd and lncli.
GO111MODULE=on go build -v -tags="dev" -o lnd-debug -ldflags "-X github.com/lightningnetwork/lnd/build.Commit=v0.5.1-beta-256-g089d461552ffc05d7b1c22d603af0fa82e788e85" github.com/lightningnetwork/lnd
go: finding github.com/ltcsuite/ltcd v0.0.0-20180701083835-cdab10132e8c
go: github.com/ltcsuite/ltcd@v0.0.0-20180701083835-cdab10132e8c: unknown revision cdab10132e8c
go: error loading module requirements
Makefile:99: recipe for target 'build' failed
make: *** [build] Error 1`

--- a/go.mod
+++ b/go.mod
@@ -34,7 +34,7 @@ require (
 	github.com/kkdai/bstream v0.0.0-20181106074824-b3251f7901ec
 	github.com/lightninglabs/neutrino v0.0.0-20181130220745-8d09312ac266
 	github.com/lightningnetwork/lightning-onion v0.0.0-20180605012408-ac4d9da8f1d6
-	github.com/ltcsuite/ltcd v0.0.0-20180701083835-cdab10132e8c
+	github.com/ltcsuite/ltcd v0.0.0-20190101042124-f37f8bf35796
 	github.com/miekg/dns v0.0.0-20171125082028-79bfde677fa8
 	github.com/rogpeppe/fastuuid v0.0.0-20150106093220-6724a57986af // indirect
 	github.com/tv42/zbase32 v0.0.0-20160707012821-501572607d02

